Solutions/Sinec Security Guard/SolutionMetadata.json

```diff
@@ -4,8 +4,8 @@
 	"firstPublishDate": "2024-07-15",
 	"providers": ["Siemens AG"],
 	"categories": {
-		"domains" : ["cybersecurity", "Cloud-based"],
-		"verticals": ["Industrial","OT","Shop Floors"]
+		"domains" : ["Security - Network"],
+		"verticals": ["NDR"]
```

Comment:
Hello @XiFneg, This is an invalid vertical. Please go through this doc 
https://github.com/MicrosoftDocs/azure-docs/blob/main/articles/sentinel/sentinel-solutions.md?msclkid=9a240b52b11411ec99ae6736bd089c4a#categories-for-microsoft-sentinel-out-of-the-box-content-and-solutions